openedx/adg/lms/applications/views.py

```diff
@@ -293,7 +293,7 @@ def get_context_data(self, request):
 
 class EducationAndExperienceView(RedirectToLoginOrRelevantPageMixin, TemplateView):
     """
-    Education and Experience View
+    View for the "My Experience" page of the written application
```

Comment:
Are we calling the second page of the written application, _My Experience_?
Better to name this Education & Experience page (as it was called before) for now, till we've changed it in the other places too. WDYT?

Reply:
We had not changed the names of the template and view to avoid future code conflicts and agreed that the existing names would suffice. However, this is a docstring, and has no potential implications on rest of the code. The reason behind calling the 2nd page of the application "My Experience" page is that it is the title of the second page as per the designs, so it makes sense to call it that. "Education & Experience page" is not accommodating enough since it does not directly cater to the background question and the references (despite the fact that background question and references are somewhat related to education and experience). Ahmed was suggesting to change the name of the template and view as well, but I didn't do that owing to the discussion we had of not changing view and file names.

Reply:
That makes sense! Thank you for filling me in.